lib/index.js

```diff
@@ -137,65 +137,91 @@ module.exports = babel => {
 			Program: {
 				enter() {
 					this.moduleConfigs = normalizeModulesConfig(this.opts.modules);
+					this.failOnError = typeof this.opts.failOnError === 'boolean' ? this.opts.failOnError : true;
+					this.logOnError = typeof this.opts.logOnError === 'boolean' ? this.opts.logOnError : true;
 					this.babel = babel;
 					this.bindings = [];
 				}
 			},
 			CallExpression(path) {
-				if (!path.parentPath.isVariableDeclarator() || !path.get('callee').isIdentifier({name: 'require'})) {
-					return;
-				}
-
-				const moduleName = path.get('arguments.0');
-				if (!moduleName.isStringLiteral()) {
-					return;
-				}
+				try {
```

Comment:
I think the try block here and in `ImportDeclaration` are not needed.  These do not actually run `html-minifier`, they just create babel bindings which are matched in `TaggedTemplateExpression`.  IMO a crash from `CallExpression` or `ImportDeclaration` always indicates a bug in `babel-plugin-html-minifier`, regardless of what source code was being processed.  I think `failOnError` and `logOnError` should apply to the minification process only.